### DISCUSSION TITLE
Output automatic scaling factor

### DISCUSSION THREAD

**Discussion by Flolaffel** [idaholab/moose#21846] [Q&A Modules: Solid mechanics] [answered]
Hello,

is there a way to output the scaling factor calculated by `automatic_scaling`?

Alternatively: What I'm doing right now is using `extra_vector_tags`, a `TagVectorAux` AuxKernel and a `NodalSum` Postprocessor to compute the reaction force on a sideset. But the result is affected by `automatic_scaling` which is why I'm looking for the scaling factor. If there is a better way than the described one that works with `automatic_scaling` enabled by default, please let me know.

**Comment by recuero** [comment, marked as answer]
IIRC there is an option in `TagVectorAux` (https://github.com/idaholab/moose/pull/20520) `scaled=false` that would return the reaction without the scaling factor. But the default behavior should be the one you are describing.

**Reply by Flolaffel** [reply]
Thanks!

**Comment by GiudGiud** [comment]
For your first question, yes, you just need to set:
verbose = true

in the executioner block to output the scaling factors

**Reply by Flolaffel** [reply]
Can only mark one answer, thank you anyway!
